### PR TITLE
Replace NetworkSettingsSelector with an explainer on why it is not relevant on Optimism for Optimism transactions.

### PR DESCRIPTION
**Pull request by 0xDaedalus**
This PR replaces the (irrelevant for Optimism) gas selector with an explanation of why editing the gas fee is not as relevant on optimism as it is on other networks.

image

Until we get an optimism-specific design of gas controls I think this is a happy medium.  Some background on why our current gas control screens are not a good fit here:
- Optimism has no mempool, so a transaction is either instantly accepted or rejected - there is no concept of regular / express / instant.
- The bulk of a transaction's gas cost is rooted in the L1 rollup fee which is not able to be edited at all by the user.
- Optimism transactions have 4 cost components - Optimism gasLimit, Optimism gasPrice, Ethereum gasLimit, and Ethereum gasPrice.

### To Test
- [ ] Attempt to edit the gas settings of an Optimism transaction - the above screen should pop up in lieu of the EIP-1559 gas controls.

### CODE REVIEW

**Comment by mhluongo** [comment]
The styling is off on Chromium

Screenshot from 2022-08-22 20-46-44

**Comment by 0xDaedalus** [comment]
I'm not able to recreate the issue on chrome and am running into lots of pain trying to get chromium running on an M1 mac.  Anyone else able to troubleshoot this?

File: ui/components/NetworkFees/NetworkSettingsSelectOptimism.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      <div className="explainer">
+        <span>
+          Gas Fees on Optimism work differently from Ethereum. The bulk of the
+          cost of an Optimism transaction is automatically deducted by the
+          network to pay the Ethereum rollup fee and is not editable by the
+          user.
+        </span>
+        <br />
+        <span>
+          Tally stays in sync with the current Optimism and Ethereum network
```
Comment (jagodarybacka) [inline ui/components/NetworkFees/NetworkSettingsSelectOptimism.tsx:28]:
```suggestion
          Tally Ho stays in sync with the current Optimism and Ethereum network
```

**Comment by 0xDaedalus** [comment]
Thanks for the feedback - should be ready for a second look:

image

**Review by jagodarybacka — APPROVED** [review]
👑

**Pull request merged**